pimd/pim_cmd.c

```diff
@@ -7227,12 +7227,12 @@ DEFUN (ip_pim_register_suppress,
 
 DEFUN (no_ip_pim_register_suppress,
        no_ip_pim_register_suppress_cmd,
-       "no ip pim register-suppress-time (5-60000)",
+       "no ip pim register-suppress-time [(1-65535)]",
        NO_STR
        IP_STR
        "pim multicast routing\n"
        "Register Suppress Timer\n"
-       "Seconds\n")
+       IGNORED_IN_NO_STR)
 {
 	char rs_default_timer[5];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
changed to destroy